linelist_investigations/report_sources/age_sex_2019-10-18.Rmd

```diff
@@ -397,14 +419,17 @@ have been generated and are in the same folder as this document):
 Explain what the exported files are using the following model:
 
 
-* `produced_rds/tab_overall.rds`: tab_overall contains the overall distributions of cases by sex.
+* `produced_rds/tab_overall.rds`: tab_overall contains the overall distributions of cases by age group and sex.

```

Comment:
Remove the whole rds export section as probably not useful for further analyses in R.

Reply:
Edited accordingly